net/nut/files/nut-monitor.init

```diff
@@ -148,7 +148,6 @@ nut_upsmon_add() {
 }
 
 build_config() {
-	local runas=nutmon
 	mkdir -m 0750 -p "$(dirname "$UPSMON_C")"
```

Comment:
This throws a warning on shellcheck: https://github.com/koalaman/shellcheck/wiki/SC2174

Doesn't seem problematic.

Reply:
ash doesn't have a problem with it...not sure we need to care about 'pure' POSIX shell.